app/grants/views.py

```diff
@@ -102,6 +102,7 @@ def get_clr_rounds_metadata():
         clr_round = CLR_ROUND_DATA['round_num']
         start_date = CLR_ROUND_DATA['round_start']
         end_date = CLR_ROUND_DATA['round_end']
+        show_bannner = CLR_ROUND_DATA['show_bannner']
```

Comment:
I think `clr_active`/`active` makes more sense - we can use this to toggle the round dates in the trust requirements banner too 👍